README.md

```diff
@@ -42,6 +42,11 @@ $ docker-compose up -d worker
 # Generate random database fixtures
 $ docker-compose run --rm api rake db:setup db:populate
 
+# (Optional) Erase local database and import copy of staging database.
+# - Only available in "development" Rails env.
+# - Ask technical team for the staging database password.
+$ docker-compose run -e STAGING_DB_PASSWORD=<...> --rm api rake db:setup db:import_staging
```

Comment:
It might be a good idea to move this to a separate section of the README. The nice thing about the way this current code block is structured is that for new people checking out the repo, they can just line-by-line copy the commands and get the app running immediately. Putting this here might cause a bit of confusion to new people.